certbot-ci/windows_installer_integration_tests/test_main.py

```diff
@@ -0,0 +1,60 @@
+import os
+import time
+import unittest
+import subprocess
+
+
+@unittest.skipIf(os.name != 'nt', reason='Windows installer tests must be run on Windows.')
+def test_it(request):
+    try:
+        subprocess.check_call(['certbot', '--version'])
+    except (subprocess.CalledProcessError, OSError):
+        pass
+    else:
+        raise AssertionError('Expect certbot to not be available in the PATH.')
+
+    try:
+        # Install certbot
+        subprocess.check_call([request.config.option.installer_path, '/S'])
+
+        # Assert certbot is installed and runnable
+        output = subprocess.check_output(['certbot', '--version'], universal_newlines=True)
+        assert 'certbot 1.' in output, 'Flag --version does not output a version.'
```

Comment:
nit: Can we use something like a regex to prevent this test from breaking when we release Certbot 2.0?

Reply:
Done.